docs/v0.3/partials/_install-sdk-go.md

```diff
@@ -0,0 +1,11 @@
+:::note
+The Dagger SDK for Go requires [Go 1.15 or later](https://go.dev/doc/install).
+:::
+
+Install the Dagger SDK for Go using the commands below:
+
+```shell
+go get go.dagger.io/dagger@v0.3.0-alpha.1
```

Comment:
Since we don't want to version this page, should we consider removing the version-specific part?

In other words:

```suggestion
go get go.dagger.io/dagger
```

Reply:
This wont work until we have a non-prerelease tag, right?